components/scream/src/physics/p3/micro_p3_iso_c.f90

```diff
@@ -331,6 +331,40 @@ subroutine cloud_sedimentation_c(kts,kte,ktop,kbot,kdir,   &
 
   end subroutine cloud_sedimentation_c
 
+  subroutine ice_sedimentation_c(kts,kte,ktop,kbot,kdir,    &
+       rho,inv_rho,rhofaci,icldm,inv_dzq,dt,odt,  &
+       qitot,qitot_incld,nitot,qirim,qirim_incld,birim,birim_incld,nitot_incld,prt_sol,qi_tend,ni_tend) bind(C)
+    use micro_p3, only: ice_sedimentation
+
+    ! arguments
+    integer(kind=c_int), value, intent(in) :: kts, kte, ktop, kbot, kdir
+
+    real(kind=c_real), intent(in), dimension(kts:kte) :: rho
+    real(kind=c_real), intent(in), dimension(kts:kte) :: inv_rho
+    real(kind=c_real), intent(in), dimension(kts:kte) :: rhofaci
+    real(kind=c_real), intent(in), dimension(kts:kte) :: icldm
+    real(kind=c_real), intent(in), dimension(kts:kte) :: inv_dzq
+    real(kind=c_real), value, intent(in) :: dt, odt
+
+    real(kind=c_real), intent(inout), dimension(kts:kte), target :: qitot
```

Comment:
Would you remind me why this interface function needs to annotate qitot, etc., as `target`?

Reply:
@ambrad so they can be added to:
```
type(realptr), dimension(num_arrays) :: vs, fluxes, qnr
```
so that they can then be shipped to generalized_sedimentation in a general way.

Reply:
Oh, right, thanks.